src/Cassandra.IntegrationTests/CommonFixtureSetup.cs

```diff
@@ -20,15 +20,13 @@ public void SetupTestSuite()
                 Trace.Listeners.Add(new TextWriterTraceListener(Console.Out));
             }
             Trace.TraceInformation("Starting Test Run ...");
-            SimulacronManager.Instance.Start();
         }
 
         [OneTimeTearDown]
         public void TearDownTestSuite()
         {
             // this method is executed once after all the fixtures have completed execution
             TestClusterManager.TryRemove();
-            SimulacronManager.Instance.Stop();
```

Comment:
We should reset this change too, along with the rest of `CommonFixtureSetup` changes. I think this is preventing for the appveyor build to finish.